deploy/crds/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -55,9 +55,6 @@ spec:
                           storageClass:
                             description: Storage class name used for Alertmanager PVC
                             type: string
-                          storageResources:
```

Comment:
This param was being used in the wrong part of the template so could never have had any effect. I think it's safe to simply remove it.